### DISCUSSION TITLE
Is index.vue always requested?

### DISCUSSION THREAD

**Discussion by Saurou** [nuxt/nuxt#19429] [Questions] [answered]
I find myself to often have index.vue and [slug].vue to be identical, rendering the same components. 
Is it possible to have only [slug].vue in root which also works as the index page?

**Comment by AloisSeckar** [comment, marked as answer]
You need double square brackets to "catch-all-including-/" route. This [makes the slug optional](https://nuxt.com/docs/guide/directory-structure/pages#dynamic-routes) and therefore also catching "/".

Demo: https://stackblitz.com/edit/github-zquzvj?file=pages/[[slug]].vue
